content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -2,48 +2,64 @@
+Medic recommends [Docker](/hosting/4.x/docker/) for most deployments. Only use [Kubernetes](/hosting/4.x/kubernetes/) for large, multi-tenant or specific hardware constrained deployments.
 
-Given all this, we recommend:
-* New, smaller production 4.x CHT deployments can use either [Docker]({{< relref "hosting/4.x/production/docker/" >}}) or [Kubernetes]({{< relref "hosting/4.x/production/kubernetes/" >}}) depending on their expected use and growth.
-* Application development for both [CHT 3.x]({{< relref "hosting/3.x/app-developer" >}}) and [CHT 4.x]({{< relref "hosting/4.x/app-developer" >}}) should use Docker Compose.
-* Production 3.x CHT deployments should use [Docker Compose]({{< relref "hosting/3.x" >}})  - Note that 3.x is [end of life]({{< relref "releases/#supported-versions" >}})  and should only be used to support existing 3.x deployments.
+Application development for both [CHT 3.x]({{< relref "hosting/3.x/app-developer" >}}) and [CHT 4.x]({{< relref "hosting/4.x/app-developer" >}}) should use Docker as well.
 
-## Kubernetes
+## Docker 
 
-Kubernetes provides advantages in managing CHT Deployments:
+Docker's has stood the test of time and has many benefits:
 
-* Resilient network across either physical or VM nodes in the cluster. This allows strong distribution of the heavy CPU and RAM loads that CouchDB can incur under heave use.
+* Easy to deploy by just downloading a compose file.
+* Trivially scales down to a 4 core laptop and [way up to a 48 core server](https://forum.communityhealthtoolkit.org/t/investigate-adding-more-shards-as-a-potential-avenue-for-improved-performance/4831). 
+* Easily integrates with existing Docker based solutions like [Watchdog](/hosting/monitoring/) or other services running in Docker.
+* Backup and restore can be achieved via simple file management via [bind mounted volumes](https://docs.docker.com/engine/storage/bind-mounts/).
+* Just enough complexity to securely and soundly run the CHT with out extra complexities of Kubernetes 
+* Natively supports one click CHT upgrades in the admin GUI
+
+Docker Compose deployment has the following services when running the CHT:
+
+* A service for each CouchDB instance.
+* A CHT API service.
+* HAProxy Healthcheck service.
+* HAProxy service.
+* Upgrade Service service.
+* A CHT Sentinel service.
+* An nginx service to act as a reverse proxy and terminate TLS connections
+
+
+## Kubernetes (k8s)
+
+{{< callout type="warning" >}}
+Only use Kubernetes on bare-metal machines with 8 cores or fewer or when dozens of CHT instances are needed in a multi-tenant deployment.
+{{< /callout >}}
+
+Given the additional knowledge needed to run and complexities to deploy, CHT on k8s has been shown to result in confusion and outages. Docker is the preferred solution for most production deployments.
+
+K8s deployments can not safely use the one click upgrade button in the admin GUI.
```

Comment:
Do you have a link to forum posts or data behind this statement to explain the why and what happens if people use that button? I think it will help understand what's lacking currently to have this possible.

Reply:
Yeah - good point!  I think this can actually be addressed (if you back up your helm charts and always use helm charts deply or restore, it works fine).  I'll just remove this for now.

fixed in 19281800